scripts/install/opt/seagate/cortx/motr/bin/motr_setup

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+
+from cortx.utils.conf_store import Conf
+from motr_mini_prov import *
+
+class Cmd:
+    """Setup Command"""
+    _index = "conf"
+
+    def __init__(self, args: dict):
+        self._url = args.url
+        Conf.load(self._index, self._url)
+        self._args = args.args
+        self._server_id = get_current_node(self)
+
+    @property
+    def args(self) -> str:
+        return self._args
+
+    @property
+    def url(self) -> str:
+        return self._url
+
+    @staticmethod
+    def usage(prog: str):
+        """Print usage instructions"""
+
+        sys.stderr.write(
+            f"usage: {prog} [-h] <cmd> <url> <args>\n"
+            f"where:\n"
+            f"cmd   post_install, config, init, test, reset, cleanup, upgarde,"
+            f" backup, restore\n"
+            f"url   Config URL\n")
+
+    @staticmethod
+    def get_command(desc: str, argv: dict):
+        """Return the Command after parsing the command line."""
+        parser = argparse.ArgumentParser(desc)
+        subparsers = parser.add_subparsers()
+        cmds = inspect.getmembers(sys.modules[__name__])
+        cmds = [(x, y) for x, y in cmds
+            if x.endswith("Cmd") and x != "Cmd"]
+        for name, cmd in cmds:
+            cmd.add_args(subparsers, cmd, name)
+        args = parser.parse_args(argv)
+        return args.command(args)
+
+    @staticmethod
+    def add_args(parser: str, cls: str, name: str):
+        """Add Command args for parsing."""
+
+        parser1 = parser.add_parser(cls.name, help='setup %s' % name)
+        parser1.add_argument('url', help='Config URL')
+        parser1.add_argument('args', nargs='*', default=[], help='args')
+        parser1.set_defaults(command=cls)
+
+
+class PostInstallCmd(Cmd):
+    """PostInstall Setup Cmd"""
+    name = "post_install"
+
+    def __init__(self, args: dict):
+        super().__init__(args)
+
+    def process(self):
+        print("Processing %s %s %s" %(self.name, self.url, self._args))
```

Comment:
Two problems:
1. In other places you use the "f-string Formatting in Python". But here, you are using the triditional format string "{} and %s". Please replace it with f-string.
2. IN other places, you use "sys.stderr.write" and "sys.stdout.write", but you are using "print" here. Can we use the same?